components/b2b/AccountHierarchy/AccountHierarchyAddForm/AccountHierarchyAddForm.tsx

```diff
@@ -235,4 +234,4 @@ const AccountHierarchyForm = (props: AccountHierarchyFormProps) => {
   )
 }
 
-export default AccountHierarchyForm
+export default AccountHierarchyAddForm
```

Comment:
Instead of holding entire selected account, just set selected account id
const [selectedParentAccountId, setSelectedParentAccountId] = useState('')
const selectedParentAccount = accounts.filter(account=> .....)

Reply:
selectedParentAccount details are required to show in company name in UI